admin/configurator.js

```diff
@@ -242,7 +242,7 @@ var loggerLevel = {
     error: true,
     debug: true
 };
-
+/*istanbul ignore next*/
```

Comment:
As these lines can't be tested, so we add specific grammar for these to improve testing rate